pr_6/javascript/random_samples/direct-response-write.js

```
@@ -0,0 +1,184 @@
+  // {fact rule=cross-site-scripting@v1.0 defects=1}
+router.get('/greeting', (req, res) => {
+    const { name } = req.query;
+    // ruleid: direct-response-write
+    res.send('<h1> Hello :' + name + "</h1>")
+})
+// {/fact}
+
+// {fact rule=cross-site-scripting@v1.0 defects=0}
+//template handle escaping
+router.get('/greet-template', (req, res) => {
+    name = req.query.name
```

Biome (2.1.2)

[error] A global variable should not be reassigned.

Assigning to a global variable can override essential functionality.

(lint/suspicious/noGlobalAssign)